### PR TITLE
Ruby on Rails Capstone Project - Budgetify app

### PR DESCRIPTION
**Pull request by codewanderlust**
> Hi, 🙋🏿‍♂️
This is a summary of what I have been able to achieve with regard to the Budgetify Rails application

I was able to accomplish the following tasks as per the requirements;

- A simple page with the name of your app and links to the sign up and log in pages.
- Sign up and log in pages
- Home page (categories page)
- For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
- When the user clicks on a category item, the application navigates to the transactions page for that category.
- There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
- For a given category, the list of transactions is presented, ordered by the most recent.
- At the top of the page the user could see the total amount for the category
- There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- The user fills out a form to create a new category, indicating their name and icon
- The user clicks the "Save" button to create the new category, and is taken to the home page on success.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- Create unit and integration tests for all the most important components of your RoR application.
- I was also able to record and walkthrough video and deployed the application on Render

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,175 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank `Microverse` for providing us the resources that enabled us to finish this challenge!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (omar-muhamad) [inline README.md]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=the%20author.%20Therefore%2C-,you%20must%20do%20it%20in%20the%20README%20of%20your%20project.,-Interactions), Kindly give credits to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) at `Acknowledgments` section.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @RoyJumah,

### Nice job, you are almost there.

giphy (2)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- [ ] N/A.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (codewanderlust) [reply README.md]: Noted, I will work on this

**Review by DammyShittu — APPROVED** [review]
Hi @RoyJumah,

## APPROVED 🟢

## Highlights
✔️ Descriptive readme

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**